src/app/analyze/stack/stack-details/stack-details.component.ts

```diff
@@ -188,10 +188,10 @@ export class StackDetailsComponent implements OnInit {
     const url: string = this.similarStacks[0].uri;
     this.recoArray[this.currentIndex]['rows'] = [];
     this.recoArray[this.currentIndex]['url'] = url;
-    for (var component in components) {
+    for (let component of components) {
```

Comment:
@joshuawilson Addressed in this commit - https://github.com/fabric8io/fabric8-ui/pull/119/commits/652c3cbec2c82e805983a7739b7e26b37931fbf9